assets/css/custom/custom-layout-buttons.css

```diff
@@ -0,0 +1,86 @@
+/* --------------------------------------
+   Layout
+   -------------------------------------- */
+
+/*
+   CSS specificity calculator: https://specificity.keegan.st/
+   Specificity components: [Inline styles] [IDs] [Classes, attributes and pseudo-classes] [Elements and pseudo-elements]
+   Specificity ranges from lowest to highest (more generic to more specific)
+*/
+
+/* Buttons - used across multiple pages */
+
+.btn-more {   /* 0 0 1 0 */
+    border-radius: 6px;
+    font-size: 18px;
+    margin: 10px 0;
+    padding: 10px 30px;
+}
+
+.btn-more-light {  /* 0 0 1 0 */
+    background-color: transparent;
+    border: 1px solid white;
+    color: white;
+}
+
+.btn-u-orange {   /* 0 0 1 0 */
+    background: #e67e22;
+}
+
+#sharethisButtons {   /* 0 0 1 0 */
+    padding-bottom: 30px;
+}
+
+.button-with-bg-image span {   /* 0 0 1 1 */
+    font-size: 24px;
+    margin: 0;
+    position: absolute;
+    display: flex;
+    display: -webkit-flex;
+    width: 100%;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    justify-content: center;
+    -webkit-justify-content: center;
+    align-items: center;
+    -webkit-align-items: center;
+    color: #fff;
+}
+
+.btn-more:hover {  /* 0 0 2 0 */
+    transition: .3s;
+}
+
+.btn-more-light:hover {  /* 0 0 2 0 */
+    background-color: white;
+    color: #687074;
+}
+
+.btn-brd.btn-u-orange {   /* 0 0 2 0 */
+    border-color: #e67e22;
+}
+
+.btn-u-orange:hover,                   /* 0 0 2 0 */
+.btn-u-orange:focus,                   /* 0 0 2 0 */
+.btn-u-orange:active,                  /* 0 0 2 0 */
+.btn-u-orange.active,                  /* 0 0 2 0 */
+.open.dropdown-toggle.btn-u-orange {   /* 0 0 3 0 */
+    background: #d35400;
+    color: #fff;
+    text-decoration: none;
+}
+
+.btn-brd.btn-u-orange:hover {   /* 0 0 3 0 */
+    color: #d35400;
+    border-color: #d35400;
+}
+
+.btn-brd.btn-u-orange.btn-brd-hover:hover {   /* 0 0 4 0 */
+    background: #d35400;
+}
+
+#sharethisButtons {   /* 0 1 0 0 */
```

Comment:
As mentioned earlier not changing the name of this selector, in this PR, maybe via another PR